src/intl/en/page-wallets-find-wallet.json

```
@@ -62,7 +62,7 @@
   "page-find-wallet-description-trust": "Trust Wallet is a decentralized multi-coin cryptocurrency wallet. Buy crypto, explore dapps, swap assets, and more while keeping control of your keys.",
   "page-find-wallet-description-unstoppable": "Unstoppable Wallet is an open-source, non-custodial storage solution known for its intuitive design and frictionless user experience. Natively integrates decentralized trading/exchange capabilities.",
   "page-find-wallet-description-zengo": "ZenGo is the first keyless crypto wallet. With ZenGo, there are no private keys, passwords or seed phrases to manage or lose. Buy, trade, earn and store Ethereum with unprecedented simplicity and safety",
-  "page-find-wallet-description-walleth": "100% open source (GPLv3) and native Android Ethereum Wallet for your service since 2017. Connect to your favorite Ethereum Applications via WalletConnect and use it directly with hardware wallets like the TREZOR or KeepKey via USB or the KeyCard via NFC.",
+  "page-find-wallet-description-walleth": "100% open source (GPLv3) and native Android Ethereum Wallet for your service since 2017. Connect to your favorite Ethereum Applications via WalletConnect and use it directly with hardware wallets.",
```

```suggestion
  "page-find-wallet-description-walleth": "100% open source (GPLv3) and native Android Ethereum Wallet for your service since 2017. Connect to your favorite dapp via WalletConnect and use it directly with hardware wallets.",
```

This could help trim a little bit more off, removing the redundant mention of "Ethereum" and using the term "dapp" instead. 

I'm fairly indifferent though, so if nobody has any further thoughts I can merge this as is, but will give it a moment for any further discussion.